tests/performance_tests/performance_test.py

```diff
@@ -26,6 +26,8 @@ class PerfTestBasicResult:
     trxExpectMet: bool = False
     basicTestSuccess: bool = False
     logsDir: str = ""
+    testStart: datetime = ""
```

Comment:
These should default to `None` since `datetime` is an object not a string.

Reply:
Addressed: [8ade49c](https://github.com/AntelopeIO/leap/pull/360/commits/8ade49c7f8211bc42226b2980bbf77a80103db5c)